WcaOnRails/app/views/competitions/_nearby_competitions.html.erb

```diff
@@ -54,8 +54,35 @@
       Last <%= Competition::NEARBY_DAYS_INFO %> days
     </label>
     <div class="col-sm-9">
-      <% nearby_count = @competition.nearby_competitions(Competition::NEARBY_DAYS_INFO, Competition::NEARBY_DISTANCE_KM_DANGER).count %>
-      <div class="<%= nearby_count < Competition::NEARBY_INFO_COUNT ? "alert alert-success" : "alert alert-danger" %>" role="alert"><%= nearby_count %> <%= "competition".pluralize(nearby_count) %> within <%= Competition::NEARBY_DISTANCE_KM_DANGER %>km and <%= Competition::NEARBY_DAYS_INFO %> days.
+      <% nearby_info = @competition.nearby_competitions(Competition::NEARBY_DAYS_INFO, Competition::NEARBY_DISTANCE_KM_DANGER).sort_by { |i| i.start_date } %>
+      <% nearby_count = nearby_info.count %>
+      <div class="<%= nearby_count < Competition::NEARBY_INFO_COUNT ? "alert alert-success" : "alert alert-danger" %>" role="alert" style="cursor: pointer;" data-toggle="modal" data-target=".bs-example-modal-sm"><%= nearby_count %> <%= "competition".pluralize(nearby_count) %> within <%= Competition::NEARBY_DISTANCE_KM_DANGER %>km and <%= Competition::NEARBY_DAYS_INFO %> days.
```

Comment:
I see `"alert ` in 2 places here, how about pulling that out to just after the `class="`?

I'm not okay with the inline css style. We don't do that here =).

As before, let's change `NEARBY_DISTANCE_KM_DANGER` to `NEARBY_DISTANCE_KM_INFO` here